vnext/Desktop.UnitTests/React.Windows.Desktop.UnitTests.vcxproj

```diff
@@ -51,6 +51,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <SDLCheck>true</SDLCheck>
+      <DisableSpecificWarnings>4267;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
>4267 [](http://example.com/codeflow?start=31&length=4)

We must not disable such warnings project-wide. Please make it local inside of the affected file.